### DISCUSSION TITLE
[4.0] Ensure all Joomla fields have overridable layouts

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#47185] [2. Cold Features]
Reminder for @wilsonge.

**Comment by tahseen-fathima98** [comment]
The code is not yet added so how can i do the changes in it ?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24490">issues.joomla.org/tracker/joomla-cms/24490</a>.</sub>

**Comment by ghost** [comment]
@zakiya27 This Issue is a Placeholder for @wilsonge as Release Lead Joomla 4, so he will work on it as he has Time.

**Comment by wilsonge** [comment]
Inputs:
https://github.com/joomla/joomla-cms/blob/4.1-dev/libraries/src/Form/Field/ComponentlayoutField.php#L46
https://github.com/joomla/joomla-cms/blob/4.1-dev/libraries/src/Form/Field/ModulepositionField.php#L136
https://github.com/joomla/joomla-cms/blob/4.1-dev/libraries/src/Form/Field/OrderingField.php#L118

Labels:
https://github.com/joomla/joomla-cms/blob/4.1-dev/libraries/src/Form/Field/NoteField.php#L57
https://github.com/joomla/joomla-cms/blob/4.1-dev/libraries/src/Form/Field/SpacerField.php#L56

**Comment by brianteeman** [comment]
Please remove the bug label

**Comment by rdeutz** [comment]
Closing this: we have this on our list anyway and it is too unspecific

**Comment by wilsonge** [comment]
There are 4 fields specifically linked here and still have not been fixed. Reopening

**Comment by brianteeman** [comment]
>  we have this on our list anyway

What list? Where?
